images/linux/scripts/tests/Tools.Tests.ps1

```diff
@@ -322,15 +316,19 @@ Describe "Containers" -Skip:(Test-IsUbuntu16) {
 }
 
 Describe "Node.js" {
-    $testCases = @("node", "grunt", "gulp", "webpack", "parcel", "yarn", "newman") | ForEach-Object { @{NodeCommand = $_} }
+    $testCases = @("node", "grunt", "gulp", "webpack", "parcel", "yarn", "newman", "netlify", "vercel") | ForEach-Object { @{NodeCommand = $_} }
 
     It "<NodeCommand>" -TestCases $testCases {
         param (
             [string] $NodeCommand
         )
 
         "$NodeCommand --version" | Should -ReturnZeroExitCode
-    }   
+    }
+    
+    It "Validate the symlink link [now]" {
```

Comment:
nit: put not to the $testCases array

Reply:
Thank you, moved `now` to the $testCases array